### PR TITLE
Activate pipeline institutional configs in template

### PR DESCRIPTION
**Pull request by jfy133**
As this should be supported now though here: https://github.com/nf-core/configs/pull/674

Meaning devs don't need to touch the section of the config, and users can add configs when they need

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 - Update python to 3.12 ([#2805](https://github.com/nf-core/tools/pull/2805))
 - Remove `pyproject.toml` from template root
 - Shorten lines in pipeline template ([#2908](https://github.com/nf-core/tools/pull/2908))
+- Permanently activated pipeline-specific institutional configs support for all pipelines ([#2936](https://github.com/nf-core/tools/pull/2936))
```
Comment (jfy133) [inline CHANGELOG.md:16]:
```suggestion
- Permanently activated pipeline-specific institutional configs support for all pipelines without need  for manual intervention ([#2936](https://github.com/nf-core/tools/pull/2936))
```

**Review by maxulysse — APPROVED** [review]

**Comment by jfy133** [comment]
@nf-core-bot fix linting

**Review by mashehu — CHANGES_REQUESTED** [review]
We should merge #2404 first I think, to not add more merge conflicts there.

**Comment by jfy133** [comment]
> We should merge #2404 first I think, to not add more merge conflicts there.

Would this effect that at all? Does the TUI pipeline creation effect the config file?

But OK

**Review by mashehu — APPROVED** [review]

**Comment by jfy133** [comment]
Woohoo1 Nice thank you all, will merge the other one on configs and :crossed_fingers: nothing breaks

**Pull request merged**